source/expressions.tex

```diff
@@ -6092,17 +6092,13 @@
 \pnum
 The \tcode{==} (equal to) and the \tcode{!=} (not equal to) operators
 group left-to-right.
-The
-lvalue-to-rvalue\iref{conv.lval},
-array-to-pointer\iref{conv.array},
-and function-to-pointer\iref{conv.func}
-standard conversions are performed on the operands.
+The operands are expected to be prvalues\iref{basic.lval}.
 The comparison is deprecated if
-both operands were of array type
-prior to these conversions\iref{depr.array.comp}.
+an array-to-pointer\iref{conv.array} conversion
```

Comment:
The array-to-pointer conversion is a single conversion -- I think "an" should be "the" here.